### ISSUE TITLE
Several problems about

### ISSUE THREAD

**Issue by liu09114** [weiliu89/caffe#176]

**Comment by weiliu89** [comment]
1. Several boxes will be randomly sampled, but only ONE is picked to crop the image.

2. data_data_0_split_* is connected to several layers in the network. For example, each priorbox layer needs to connect to data

3. You mean AnnotatedDataLayer? It will use the original image.

4. Confidence is the raw value from mbox_conf. Probability is applying softmax on top of it.

**Comment by liu09114** [comment]
@weiliu89  Thanks your reply.
2、When training, the data layer has batch_size=8, does this mean we sample 8 patches from 8 different images?  Do all the  data_data_0_split_*  share the same data? I can't see any split layer, where the split operation is assigned? 
3、Yes, I mean the AnnotatedDataLayer.
4、 The SSD computes the probability? The test.prototxt shows：
layer {
  name: "mbox_conf_softmax"
  type: "Softmax"
  bottom: "mbox_conf_reshape"
  top: "mbox_conf_softmax"
  softmax_param {
    axis: 2
  }
}
